01-16-2025 Update:

ragustinesantos opened the pull request:
- Schema for Employee, Item, and Order Requisition has been updated to include an isActive boolean value
- Schema for Order Requisition has been updated to include an isComplete boolean value
- The respective api routes for Employee, Item and Order requisition have been updated to reflect the added values
- Created odor database service and api routes
- Updated ror and ror template patch api routes to include .strict() in the schema for parsing.

ragustinesantos committed:
01-16-2025 Update:
- Schema for Employee, Item, and Order Requisition has been updated to include an isActive boolean value
- Schema for Order Requisition has been updated to include an isComplete boolean value
- The respective api routes for Employee, Item and Order requisition has been updated to reflect the added values
- Created odor database service and api routes
- Updated ror and ror template patch api routes to include .strict() in the schema for parsing.

ragustinesantos committed:
01-16-2025 Update:
- Updated Line 23 in odor-service.tsx from rorId to odorId

markusluisflores commented on app/_utils/schema.tsx:
```diff
@@ -40,6 +42,7 @@ export const defaultItem: Item = {
   isCritical: false,
   isCriticalThreshold: 0,
   minPurchaseQty: 0,
+  isActive: false,
```
I think it would be better if we default this to 'true' since we use this for new items

markusluisflores reviewed (changes requested):
The changes look good to me aside from the one line I commented on.

ragustinesantos committed:
01-17-2025 Update:
- Changed default item isActive value to true

ragustinesantos committed:
01-17-2025 Update:
- Updated OnDemandOrderToEdit interface and its corresponding api methods to exclude odorId
- Removed the NewItemOrderToEdit interface since the objects will now be stored within the OnDemandOrder objects and will not be requiring its own database service
